utils/mock-data-loader/setup-deletion-audit-logs.ts

```diff
@@ -0,0 +1,94 @@
+import { ObjectId } from 'mongodb';
+import { MONGO_DB_COLLECTIONS } from '@ukef/dtfs2-common';
+import { getCollection, getConnection } from './database/database-client';
+import { logger } from './helpers/logger.helper';
+
+export const deleteDeletionAuditLogsCollection = async () => {
+  logger.info('Dropping deletion audit logs collection');
+  const collection = await getCollection(MONGO_DB_COLLECTIONS.DELETION_AUDIT_LOGS);
+
+  try {
+    await collection.drop();
+  } catch {
+    // This will throw an error if the collection doesn't exist
```

Comment:
This can probably be done in a better way by asserting if `collection` is defined